docs/baking.md

```diff
@@ -94,9 +98,9 @@ you should init/update config in the data directory to match the desired network
 before importing the snapshot. To do that, run one of the following commands
 (depending on whether the config in the given data directory was initialized previously):
 ```
-sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+sudo -u tezos tezos-node-<network> config init --network <network>
```

Comment:
from user's perspective --netwrok is redundant here... can alias also include `--network`? alternatively, perhaps tezos-node-<network> should be a script?

Reply:
Indeed, including `--network` to the `tezos-node-<networks>` scripts sound like a good way to go

Reply:
Actually I think we should rather remove this snapshot troubleshooting section entirely.

We now call `config init` as part of the package installation process, so the problem that could have caused this is no longer there if one follows the guide.

It can still happen if a user decides to manually recreate/alter the node data dir, but those are not the kind of problems that packaging should solve IMO.

So, the `--network` option is not needed by other `tezos-node` commands as the `network` is already specified in the config file **and** we already generate the correct config for each network by default.
I think it's fair to say that we can avoid complicating the scripts to support solving a problem that we now prevent.